pwiz_tools/Skyline/Model/DdaSearch/MsFraggerSearchEngine.cs

```diff
@@ -342,7 +342,7 @@ private void EnsureFastaHasDecoys()
 
             // ReSharper disable LocalizableElement 
             IEnumerable<string> commonAffixes = new [] { "decoy", "dec", "reverse", "rev", "__id_decoy", "xxx", "shuffled", "shuffle", "pseudo", "random" };
-            commonAffixes = commonAffixes.Concat(commonAffixes.Select(o => o.ToUpperInvariant()));
+            commonAffixes = commonAffixes.Concat(commonAffixes.Select(o => o.ToUpperInvariant())).ToArray();

```

Comment:
Strange that it should need ToArray() for an IEnumberable<string>, but I am sure there are no perf issues in this case.

Reply:
Yeah, I guess "possible" is the keyword in "warning: possible multiple enumeration" but no harm in playing it safe here, I think.